scipy/stats/tests/test_stats.py

```diff
@@ -3493,7 +3544,6 @@ def test_ttest_ind():
         assert_equal(stats.ttest_ind(anan, np.zeros((2, 2))),
                      ([0, np.nan], [1, np.nan]))
 
-
```

Comment:
Don't delete this line--module-level `def`s should have two blank line before them (PEP 8).